partcad/tests/unit/test_cache_hash.py

```
@@ -0,0 +1,53 @@
+#
+
+"""The cache format version takes part in every hash, and only that."""
+
+import hashlib
+
+from partcad.cache_hash import VERSION, CacheHash
+
+
+def test_version_is_hashed_before_the_data():
+    """A bump of VERSION has to move every entry to a new key."""
+    hash = CacheHash("test", cache=True)
```

Ruff (0.16.0)

[error] Variable `hash` is shadowing a Python builtin

(A001)